### ISSUE TITLE
Shift TsdFrame timestamps

### ISSUE THREAD

**Issue by dlevenstein** [pynapple-org/pynapple#227]
Is there a reason why randomize.shift_timestamps doesn't work for a TsdFrame? Seems like it should follow the same premise and would be useful for the ability to circularly shift timestamps of continuous data.

**Comment by dlevenstein** [comment]
I guess it would be shifting the data, not the timestamps

**Comment by gviejo** [comment]
Yes that sounds like a different function?

**Comment by dlevenstein** [comment]
Yeah, but I great one to have for permutation tests with randomize!

**Comment by gviejo** [comment]
Hey I don't think this is in the roadmap yet. Do you want to open a PR to make it happen or can I close this issue?
